### DISCUSSION TITLE
[Bug]: Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "application/octet-stream". Strict MIME type checking is enforced for module scripts per HTML spec.

### DISCUSSION THREAD

**Discussion by kamaludin21** [beyondcode/herd-community#614] [General]
### Platform

Windows

### Operating system version

Windows 11 23H2

### System architecture

Intel (x86)

### Herd Version

1.0.1

### PHP Version

8.3.4

### Bug description

Error appear when work with javascript library that need MIME type of "application/octet-stream". i have found this issues at my server, it is nginx config problem. the error message appear is:

 Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "application/octet-stream". Strict MIME type checking is enforced for module scripts per HTML spec.

### Steps to reproduce

error when try to load file pdf using library mozilla/pdf.js

### Relevant log output

```shell
pdf.mjs:1 -> Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "application/octet-stream". Strict MIME type checking is enforced for module scripts per HTML spec. 

viewer.mjs:1 -> Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "application/octet-stream". Strict MIME type checking is enforced for module scripts per HTML spec.
```

**Comment by mpociot** [comment]
Which framework do you use?

**Reply by kamaludin21** [reply]
thank's for reply, 

i try with laravel 11. but, i've solved it by adding mjs type in nginx.conf (\.config\herd\config\nginx)

`application/javascript                           js mjs;`

this is okay?
